.rubocop_todo.yml

```diff
@@ -0,0 +1,23 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config --auto-gen-only-exclude --exclude-limit 99999`
+# on 2023-05-12 18:20:53 UTC using RuboCop version 1.50.2.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 9
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: have_no, not_to
+Capybara/NegationMatcher:
+  Exclude:
+    - 'spec/system/demo_splash_spec.rb'
```

Comment:
when i updated the spec to follow this cop, the spec failed because it didn't wait for the text to go away. I think we want to update this to support `have_no` in our `betterlint` config